memdown.js

```diff
@@ -2,12 +2,12 @@ var inherits          = require('inherits')
   , AbstractLevelDOWN = require('abstract-leveldown').AbstractLevelDOWN
   , AbstractIterator  = require('abstract-leveldown').AbstractIterator
   , ltgt              = require('ltgt')
-  , setImmediate      = global.setImmediate || process.nextTick
+  , setImmediate      = require('immediate')
   , createRBT = require('functional-red-black-tree')
   , globalStore       = {}
 
 function toKey (key) {
-  return typeof key == 'string' ? '$' + key : JSON.stringify(key)
+  return '$' + key
```

Comment:
Although I guess it may technically be a breaking change, although I've never heard of this feature being documented anywhere in the level ecosystem (`levelup({foo: 'bar'})`?) so I'd consider it dubious.